src/components/UserRewardDisplay.tsx

```diff
@@ -1,9 +1,31 @@
-import { PropsWithChildren } from "react";
-import { IUserReward } from "../models/User";
+import { PropsWithChildren, useState } from "react";
+import { IReward, IUserReward } from "../models/User";
+import { RedeemPopup } from "./RedeemPopup";
+import RewardCard from "./RewardCard";
+
+function UserRewardDisplay({ user_rewards }: PropsWithChildren<{ user_rewards: Array<IUserReward> }>) {
+  const [current_reward, set_current_reward] = useState({} as IReward);
+  const [show_reward_popup, set_show_reward_popup] = useState(false);
+  const handle_click = (reward: IReward) => {
+    return () => {
+      set_current_reward(reward);
+      set_show_reward_popup(true)
```

Comment:
Add semicolon.

Why is it necessary to return the useStates rather than simply set them?